scripts/export_uml_json.py

```diff
@@ -0,0 +1,146 @@
+"""
+
+Convert a umls release to a json file of concepts.
+
+"""
+import sys
+import json
+import plac
+
+# preferred definitions (from S2)
+DEF_SOURCES_PREFERRED = {'NCI_BRIDG', 'NCI_NCI-GLOSS', 'NCI', 'GO', 'MSH', 'NCI_FDA'}
+
+
+@plac.annotations(
```

Comment:
Please could you switch to `argparse`, to be consistent with the rest of the codebase. (`init_model` is an exception because it is copied almost verbatim from the spacy command and will be replaced soon)

Reply:
done